PokeAlarm/Manager.py

```diff
@@ -73,6 +74,12 @@ def __init__(self, name, google_key, locale, units, timezone, time_limit, max_at
         self.__queue = multiprocessing.Queue()
         self.__process = None
 
+        # Update cp ranges
+        with open(get_path('data/cp_ranges.json'), 'r') as f:
```

Comment:
Can this move be moved to Utils.py? Seems like it's universal for all managers

Reply:
Totally should, yes